File: Projects/rosalind/SSET.java
import java.io.*;
import java.util.*;
import java.math.*;

public class SSET {
	public static void main (String[] args) throws IOException {
		final Scanner in = new Scanner(System.in);
		System.out.println(new BigInteger("2").pow(in.nextInt()).mod(new BigInteger("1000000")));
	}
}

File: Projects/rosalind/SSET.kt
package rosalind

import java.io.IOException
import java.math.BigInteger
import java.util.*

object SSET {
    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        println(BigInteger("2").pow(stdin.nextInt()).mod(BigInteger("1000000")))
        stdin.close()
    }
}